fisher.fish

```diff
@@ -56,15 +56,21 @@ function $fisher_cmd_name -d "fish plugin manager"
 
     set -g fisher_version "2.12.0"
     set -g fisher_spinners ⠋ ⠙ ⠹ ⠸ ⠼ ⠴ ⠦ ⠧ ⠇ ⠏
-    set -g __fisher_stdout /dev/stdout
-    set -g __fisher_stderr /dev/stderr
+
+    if [ "$PREFIX" = "/data/data/com.termux/files/usr" ]
```

Comment:
I could instead check if the file exists (`[ - f /dev/stderr ] `), that would be cleaner, but I'm not sure how to detect the man version. I thought this is simple, pretty safe and has a minimum risk on other platforms but I agree checking the file existence would be better. I'll update the PR. 

I'll comment where this condition would be "more problematic" separately.